.github/CONTRIBUTING.md

```diff
@@ -136,6 +136,11 @@ This is the optimal workflow that should be followed when managing a new PR:
       arise which could require these rules to be broken or bent. If you have any
       doubts regarding the best route to take just ask and some collaborator will
       help you;
+11. If the PR has been merged to a branch other than the default `master` branch
+    and it will be a noticeable length of time before that branch is merged to
+    `master`, then the `solved` tag should be added to the PR to make it clear at
+    a glance that nothing else needs doing for that PR in order for it to
+    auto-close.
```

Comment:
Didn't we verify in our email exchange that PRs get automatically closed when the code is merged in their target branch and not when merged in the default/master branch?

In fact when I added the "solved" label I thought of using it together with the "bug" label marking:

- bug-tracking issues once the bug has been solved (in other words "solved" is the opposite of "wont-fix")
- PRs solving a bug

Reply:
Oops, that was just a wording error due to late-night lack of caffeine. I meant to say that point for **issues** (which don't auto-close until stuff hits default branch), not for **PRs** (which do). I was inspired by seeing all the issues presently sitting open until `development` is merged into `master` (you probably noticed I tagged a lot of them `solved`). Because you prefer that `wont-fix` is mutually exclusive with `solved` (as opposed to being an additional "solution-detail") then I will change that wording: e.g.:

> ...then either the `solved` or `wont-fix` label should be added to the issue...

and will untag the present issues with `wont-fix`. Will do that in a minute.

Reply:
Regarding the PR/issue mismatch: ok

Regarding the labels I'd use "solved" as I explained, while we could introduce a "waiting-merge" or something similar to do what you describe.

Reply:
...but unless I've misunderstood, there would then never be a need for a separate "waiting-merge" tag, because:
* a PR gets auto-closed on merge, so it will never need to be tagged "solved" because it will already be closed (and semantically, the PR is never "solved", issues are "solved" by commits/PRs)
* an issue will get auto-closed when a PR which "solves" it is merged to master (so it would make sense to label the issue "solved" when the PR is still only merged to target-branch)
* issues will always therefore be either "unsolved", "wontfix", or "solved" - making "waiting-merge" redundant

Reply:
...of course "unsolved" is not a tag, but is the lack of a "solved" or "wontfix" tag.

Reply:
Yes, you're right.  
So we have these "label names": meaning associations:

- "bug": this issue/PR regards a bug in the code
- "solved": the bug this issue/PR is about has been solved (if the issue itself is closed this means the code is now merged into master, otherwise the code is waiting to be merged into master)
- "wontfix": the bug in this issue/PR is about cannot/will not be resolved
- neither "solved" nor "wontfix": the bug this issue/PR is about is still being worked on